decentraland/App.js

```diff
@@ -61,11 +65,23 @@ Is the config file correct?`;
     const account = await retrieveAccount();
     if (account) {
       store.dispatch(setAccount(account));
-      const creationStatus = await retrieveAccountCreationStatus();
-      if (creationStatus)
+      let creationStatus = await retrieveAccountCreationStatus();
+      let creationActions = await retrieveAccountCreationActions();
+
+      const isOldAccount =
+        account !== null && creationStatus === null && creationActions === null;
+
+      if (isOldAccount) {
+        ({
+          creationStatus,
+          creationActions,
+        } = await restoreCreationStateOfAccountFromBlockchain(account));
```

Comment:
We should test if this method works by checking out an old commit and then running this code

Reply:
I've made this test here setting `creationStatus` and `creationActions` to `null`

Reply:
Yep, I agree it ought to work in theory - just thought maybe we should be paranoid since we have some users out there now

Reply:
Ok, I got it

Reply:
. Clear app data
. Checkout from commit `ca55a44be00e74a5fa79927949a980fc0af11d02`
. Start app
. MyAccountScreen shows 0%
. Account created
. MyAccountScreen shows 100%
. Restart app
. MyAccountScreen shows 25%
. Checkout from branch `feature/25-percent`
. Start app
. MyAccountScreen shows 25% :x: 

Trying to figured out

Reply:
Hmm, weird. Okay, thanks. Maybe add some temporary log statements throughout the code base?

Reply:
Just fixed.

Reply:
Woo! Great job.

Reply:
Should we include a test that catches this case or are we all set as-is?

Reply:
Good idea

Reply:
I tried the steps above out to be double-paranoid, and I can confirm it's showing 100% for me at the end as it should

Reply:
Done on PR https://github.com/tasitlabs/tasit/pull/325

Reply:
Resolving this